P5/Source/Specs/model.annotationBodyLike.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Copyright TEI Consortium. 
+Dual-licensed under CC-by and BSD2 licences 
+See the file COPYING.txt for details
+$Date$
+$Id$
+-->
+<?xml-model href="http://jenkins.tei-c.org/job/TEIP5-dev/lastSuccessfulBuild/artifact/P5/release/xml/tei/odd/p5.nvdl" type="application/xml" schematypens="http://purl.oclc.org/dsdl/nvdl/ns/structure/1.0"?>
+<classSpec xmlns="http://www.tei-c.org/ns/1.0" module="tei" type="model" ident="model.annotationBodyLike">
```

Comment:
The name is incorrect (as there is no `<annotationBody>` element, at least not yet :-). Either “model.annotationBodyPart” (following the pattern of “divBottomPart” and “editorialDeclPart”) or  “model.annotationPart.body” (following the pattern of  “entryPart.top” and “publicationStmtPart.agency”).

I think, per AB-namecon, there is more on the side of “model.annotationPart.body”.

Reply:
model.annotationPart.body seems more correct to me.

Reply:
Check. As far as I’m concerned, rename away.
Let me know if you want me to do the `<desc>`s (above).